web/projectsEvaluation.php

```diff
@@ -394,13 +394,41 @@ function loadProjects() {
             text: 'Extended view',
             handler: showExtendedView,
         }],
+        initComponent: function() {
+            this.tbar = this.buildTopToolbar();
+            Ext.ux.ExportableGridPanel.superclass.initComponent.apply(this, arguments);
+        },
+        buildTopToolbar: function () {
+            return [{
+                text: 'Details',
+                id: this.id + 'DetailsBtn',
+                ref: '../detailsBtn',
+                disabled: true,
+                iconCls: this.iconCls + '-go',
+                handler: this.onDetails,
+                scope: this
+            }]
+        },
```

Comment:
Instead of returning this array and assigning it to the `tbar` property in line 398, do that assigment here instead:
```
tbar: [{
    text: 'Details',
    ...
}],
```
You will have to remove any references to `this`:
* iconCls: use the string it refers to.
* handler: see the comment below.
* scope: I think you don't need it.